### PR TITLE
Added fetch to navigation menu submenu shopping cart

### PR DESCRIPTION
**Pull request by ReinisZogots**

### CODE REVIEW

File: src/components/molecules/navigationMenu/index.tsx [hunk 1]
```diff
@@ -1,23 +1,17 @@
 import React, { useEffect, useRef } from "react";
-import Structure from "../../../structure.json";
 import "./style.scss";
-import { SubMenuItem } from "../../organisms/navigation";
+import { MenuItem, SubMenuItem } from "../../../renome.service";
+import { NavigationIcons } from "../../../renome.service";
```
Comment (Jalal999) [inline src/components/molecules/navigationMenu/index.tsx:4]: you could combine this line with previous one.
import { MenuItem, SubMenuItem, NavigationIcons } from "../../../renome.service";

File: src/components/molecules/navigtionSubmenu/index.tsx [hunk 2]
```diff
@@ -1,16 +1,17 @@
 import React, { useEffect, useRef } from "react";
-import Structure from "../../../structure.json";
-import { SubMenuItem } from "../../organisms/navigation";
+import { NavigationIcons } from "../../../renome.service";
+import { SubMenuItem } from "../../../renome.service";
```
Comment (Jalal999) [inline src/components/molecules/navigtionSubmenu/index.tsx:3]: combine with previous line

**Review by ilze-dom — APPROVED** [review]

**Review by Jalal999 — APPROVED** [review]

File: src/components/organisms/navigation/index.tsx [hunk 3]
```diff
@@ -1,18 +1,22 @@
 import React, { useState } from "react";
 import logo from "../../../assets/images/logo.svg";
-import Structure from "../../../structure.json";
 import Menu from "../../molecules/navigationMenu";
 import Cart from "../../molecules/shoppingCart";
 import Submenu from "../../molecules/navigtionSubmenu";
+import {Item, MenuItem, NavigationIcons, SubMenuItem,} from "../../../renome.service";
```
Comment (roman497) [inline src/components/organisms/navigation/index.tsx:6]: No needl last `,`

File: src/components/molecules/navigationMenu/index.tsx [hunk 4]
```diff
@@ -1,23 +1,16 @@
...
+  menuItems: MenuItem []
 }
 
-export interface MenuItem {
-  title: string,
-  url?: string,
-  isSubMenu?: boolean,
-  subMenu?: SubMenuItem[]
-}
+const Menu: React.FC<MenuProps> = ({ toggleSubMenu, setIsMenuOpen, navigationIcons, menuItems }) => {
```
Comment (roman497) [inline src/components/molecules/navigationMenu/index.tsx:12]: types?

**Pull request merged**